roles/openstack-meta/defaults/main.yml

```diff
@@ -445,3 +445,101 @@ openstack_meta:
           config_dirs: /etc/heat
           config_files: /etc/heat/heat.conf
           restart: on-failure
+  mongodb:
+    services:
+      mongodb_server:
+        ubuntu:
+          name: mongodb-org-server
+          desc: MongoDB Server
+        rhel:
+          name: mongodb-server
```

Comment:
we need to tune this systemd config. How do we want process to be started? We should look at out of the box systemd and try to match it.  Commen goes for other systemd defn (aodh, ceilometer) as well.